### DISCUSSION TITLE
关于caddy的反向代理的问题

### DISCUSSION THREAD

**Discussion by moemoequte** [AlistGo/alist#1498] [Q&A]
alist版本(使用的是稳定版的docker镜像)
```
Built At: 2022-08-13 03:55:32 +0000
Go Version: go1.18.5 linux/amd64
Author: Noah Hsu <i@nn.ci>
Commit ID: a66b2d9
Version: v2.6.4-0-ga66b2d9
WebVersion: 2.6.4
```
我的caddy配置是这样的
```
cialloo.com, www.cialloo.com {
    root * /var/www/Web/www/public
    handle_path /phpmyadmin/* {
        reverse_proxy 127.0.0.1:82
    }
    handle_path /alist/* {
        reverse_proxy 127.0.0.1:87
    }
    file_server
}
```
我的phpmyadmin docker镜像可以正常使用, 但是打开```https://www.cialloo.com/alist/```整个页面都是空白的, 谷歌浏览器和Edge浏览器的情况都是一片空白, ```handle_path /alist*``` 和 ```handle /alist/*``` 和 ``` handle alist*```都试过  
像下面这样配置却是可以正常显示
```
cloud.cialloo.com {
    reverse_proxy 127.0.0.1:87
}
```

**Comment by dijkexo** [comment]
可以这样试试：

`reverse_proxy /alist* 127.0.0.1:87`

**Comment by UnforgetMemory** [comment]
1.将[配置文件](https://alist.nn.ci/zh/config/configuration.html)中的 `site_url `赋予 【整个反代路径的值】；
2.例如`https://www.baidu.com/alist` 或者 `https://www.baidu.com` ；
3.鉴于你的caddy配置，可设置为：`https://www.cialloo.com/alist` ;

**Reply by UnforgetMemory** [reply]
个人观察原因：反代了二级目录但是js请求的是父级目录
配置文件中也有介绍用途
